src/main/resources/cli-help/ballerina-dist-remove.help

```diff
@@ -3,22 +3,21 @@ NAME
 
 SYNOPSIS
        bal dist remove <distribution>
-       bal dist remove <-a | --all>
 
-DESCRIPTION
-       Removes the specified Ballerina distribution from your local environment.
-
-       However, you are not allowed to remove the distribution, which is currently active.
+OPTIONS
+       -a, --all
+          Remove all the Ballerina distributions
```

Comment:
It will remove all the Ballerina distributions except the currently active one from your local machine.